routes/article.js

```diff
@@ -94,12 +94,20 @@ function updateNews() {
     })
 }
 
+function getAmount(vote) {
+    return Math.abs(vote) / vote || 0
+}
+
 router.post('/vote', (req, res) => {
     console.log(`votes modified by ${req.body.vote} for ${req.body.id}`)
     console.log(req.body)
-    let amount = parseInt(req.body.vote, 10)
+    let amount = getAmount(req.body.vote)
+    console.log(amount)
+    if (amount === 0) {
+        return false
```

Comment:
For errors you should do as so:
```javascript
res.status(500).send({
  error: "Recieved invalid number for vote",
  reqBody: req.body
})
return
```
When you don't respond, the client is left without information of what has happened, and awaits for the response